### PR TITLE
Add missing settings translations

### PR DESCRIPTION
**Pull request by simoebenhida**

### CODE REVIEW

**Comment by fletch3555** [comment]
5 languages huh? You're either quite the linguist, or proficient with translate.google.com

**Comment by simoebenhida** [comment]
LOL i speak french the other ones were missing and i just worked with google translation.
btw my native language is arabic and like i saw in translation file almost all is not right (just a google translation) So you need to thank me instead for adding this.

File: publishable/lang/fr/voyager.php [hunk 1]
```diff
@@ -130,9 +130,15 @@
         'successfully_deleted' => 'Paramètres supprimés avec succès',
         'already_at_top'       => 'Déjà en haut de la liste',
         'already_at_bottom'    => 'Déjà en bas de la liste',
+        'key_already_exists'   => 'La clé :key déjà existe',
```
Comment (Julien-Git1) [inline publishable/lang/fr/voyager.php:133]: La clef : key existe déjà

File: publishable/lang/fr/voyager.php [hunk 2]
```diff
@@ -130,9 +130,15 @@
...
         'moved_order_up'       => 'Trier le paramètre :name en ordre croissant',
         'moved_order_down'     => 'Trier le paramètre :name en ordre décroissant',
         'successfully_removed' => 'Valeur :name supprimée avec succès',
+        'group_general'        => 'Général',
+        'group_admin'          => 'Administrateur',
+        'group_site'           => 'Site',
+        'group'                => 'Groupe',
+        'help_group'           => 'Groupe ce paramètre est affecté à',
```
Comment (Julien-Git1) [inline publishable/lang/fr/voyager.php:141]: 'help_group'           => 'groupe auquel ce paramètre est attribué',

**Review by Julien-Git1 — APPROVED** [review]

**Review by marktopper — APPROVED** [review]
Checked the translations using Google translate. Might be better than *not* having the translations.

**Pull request merged**